server/controllers/offer/getOfferDetails.js

```diff
@@ -1,3 +1,8 @@
-module.exports = (req, res) => {
-  res.send(' get offer details ');
+const { getOffers } = require('./../../database/queries/offers/getOfferId');
```

Comment:
@AhmedAl-Almi I have a minor naming suggestion: rename this function to `getOffer`, so that everyone knows that it only returns one offer.